core/packages/test/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@snowbridge/test",
+  "version": "1.0.0",
+  "description": "Snowbridge Integration Tests",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Snowfork/snowbridge.git",
+    "directory": "core/packages/test"
+  },
+  "scripts": {
+    "test:integration": "mocha --timeout 600000 --exit"
```

Comment:
Should we add eslint to our tests as well?

Reply:
First priority is to convert them to typescript. Then eslint :)